README.md

```diff
@@ -6,8 +6,53 @@ How to start the floorIsJava application
 1. Run `mvn clean install` to build your application
 1. Start application with `java -jar target/FloorIsJava-1.0-SNAPSHOT.jar server config.yml`
 1. To check that your application is running enter url `http://localhost:8080`
+2. To see your application's Swagger UI http://localhost:8080/swagger
 
-Health Check
+Database
 ---
+Create an empty database: 
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
+Run the following command from the base of your cloned directory to create the required database structure:
+`mysql --host=<localhost> --user=<your_username> --password=<your_password> <your_database_name> < employeesdb.sql`
+
+Config
+---
+The following environment variables need to be set to enable database connection:
+
+DB_USERNAME
+
+DB_PASSWORD
+
+DB_HOST
+
+DB_NAME
+
+
+
+Tests
+---
+Run `mvn clean test` to run unit tests
+
+Run `mvn clean integration-test` to run integration tests (this may require VPN for database access)
+NOTE: Integration tests require connection to the database which may require VPN
+
+Build and run the service through docker
+
+You can build the api in a number of ways using docker and integrate it with a database, these are listed below: pre-requisite = docker and docker compose are installed in your local system.
+
+Ensure the environment variables are correct for your chosen db or enter these as additional arguments on the docker build command.
+
+
+Run "docker build <service name given + optional tag> ." from your src directory. This will read from your docker file, build the environment required for the image, build your service and create the image locally.
```

Comment:
you also need to provide params when doing docker build

Reply:
added now